test/find-owners.js

```diff
@@ -40,24 +40,46 @@ suite('integration', function() {
     });
 
     test('parent directory contains file', function() {
-        return findOwners(['2dcontext/scroll'])
+        return findOwners(['dom/nodes/attributes.html'])
           .then(function(reviewers) {
               reviewers.sort();
 
               assert.deepEqual(reviewers, [
-                  'AmeliaBR', 'annevk', 'fserb', 'jdashg', 'kenrussell'
+                  'annevk', 'jdm', 'zqzhang'
               ]);
           });
     });
 
     test('consolidation of multiple directories', function() {
-        return findOwners(['resources', '2dcontext'])
+        return findOwners(['resources/testharness.js', 'dom/nodes/attributes.html'])
           .then(function(reviewers) {
               reviewers.sort();
 
               assert.deepEqual(reviewers, [
-                  'AmeliaBR', 'annevk', 'ayg', 'fserb', 'gsnedders',
-                  'jdashg', 'jgraham', 'kenrussell'
+                  'annevk', 'ayg', 'gsnedders',
+                  'jdm', 'jgraham', 'zqzhang'
+              ]);
+          });
+    });
+
+    test('only WEB_FEATURES.yml changes', function() {
+        return findOwners(['dom/WEB_FEATURES.yml'])
+          .then(function(reviewers) {
+              reviewers.sort();
+
+              assert.deepEqual(reviewers, [
+                  'foolip', 'jcscottiii'
+              ]);
+          });
+    });
+
+    test('WEB_FEATURES.yml and test changes', function() {
+        return findOwners(['dom/WEB_FEATURES.yml', 'dom/nodes/attributes.html'])
+          .then(function(reviewers) {
+              reviewers.sort();
+
+              assert.deepEqual(reviewers, [
+                  'annevk', 'jdm', 'zqzhang'
```

Comment:
In other words, if a PR touches both `WEB_FEATURES.yml` and tests, it gets routed only to the regular reviewers. I think this is best for the long term, that we don't expect web-features maintainers to review every such change. We just want to reduce the noise from changes that are purely web-features.

Reply:
This makes sense for now. And we can always improve the linting (for example, linting based on the vendored data.json) to help regular reviewers have confidence in those changes.

Reply:
Yeah, that would be good even if you and I are the only reviewers, because we'll never catch all minor typos :)